lib/sdf-server/src/server/service/change_set/abandon_change_set.rs

```diff
@@ -28,10 +28,14 @@ pub async fn abandon_change_set(
     Json(request): Json<AbandonChangeSetRequest>,
 ) -> ChangeSetResult<()> {
     let mut ctx = builder.build_head(access_builder).await?;
-
+    let maybe_head_changeset = ctx.get_workspace_default_change_set_id().await?;
+    if maybe_head_changeset == ctx.get_workspace_default_change_set_id().await? {
```

Comment:
Won't this always fail?? You are setting the variable to the default changeset if and then checking if it's the default changeset

Don't you want to check against the current changeset??